README.md

```diff
@@ -127,11 +127,27 @@ machine. These are stored in the "refs/notes/devtools/discuss" ref, and
 annotate the first revision in the review. They must conform to the
 [comment schema](schema/comment.json).
 
-## Plugins
+## Integrations
+
+### Libraries
+
+  - [Go (use git-appraise itself)](https://github.com/google/git-appraise/blob/master/review/review.go)
+  - [Rust](https://github.com/Nemo157/git-appraise-rs)
+
+### Graphical User Interfaces
+
+  - [Git-Appraise-Web](https://github.com/google/git-appraise-web)
```

Comment:
Sorry I forgot to do that, and thanks for pointing it out.

It seems like I had a bunch of repositories where I forgot to enable the issue tracker, so catching that really helped me out.